scripts/init.sh

```diff
@@ -4,9 +4,12 @@ set -e
 
 echo "*** Initialising WASM build environment"
 
-rustup update nightly
+if [ -z $CI_PROJECT_NAME ] ; then
```

Comment:
@pepyakin basically we just want to avoid running rustup while in a rust container as it will cause an os link error and bomb out.

So this just checks if it's running via gitlab and avoids running rustup in that case.